### PR TITLE
Add blog, CNCF glossary Traditional Chinese is online for the first time

### PR DESCRIPTION
**Pull request by ydFu**
CNCF glossary official website, Traditional Chinese is online for the first time! Hot debut!(2023/11/03)

### CODE REVIEW

**Comment by ydFu** [comment]
cc @hwchiu @pichuang @tico88612

**Comment by ching-kuo** [comment]
@ydFu please fix the CI check

File: blog/glossary-zh-tw-first-published-2023.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+title: "重磅消息: CNCF glossary 官方網站，繁體中文首次上線!火熱登場!📣📣📣"
+description: "CNCF glossary official website, Traditional Chinese is online for the first time! Hot debut!(2023/11/03)"
+authors: ydFu
```
Comment (ching-kuo) [inline blog/glossary-zh-tw-first-published-2023.md:4]: Author not added to author file

Comment (ydFu) [reply blog/glossary-zh-tw-first-published-2023.md:4]: @ching-kuo 
Thanks for the suggestion

File: blog/glossary-zh-tw-first-published-2023.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+date: 2023-11-07T18:00
+---
+
+![glossary-zh-tw website](/img/cncf-glossary-zh-tw.png)
+
+## 重磅消息: CNCF glossary 官方網站，繁體中文首次上線!火熱登場!📣📣📣(2023/11/03)
+大家可以直接連上 CNCF glossary: https://glossary.cncf.io/zh-tw/
+，針對雲端原生詞彙進行查閱，目前首次上線總共有 44 個 CNCF 繁體中文內容，包含 Kubernetes, DevOps 等雲端原生內容。
+
+## 專案介紹:
```
Comment (tico88612) [inline blog/glossary-zh-tw-first-published-2023.md:16]:
```suggestion
## 專案介紹
```

File: blog/glossary-zh-tw-first-published-2023.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+- [Kay Lin](https://github.com/kaiyee0)
+- [Kehao Chen](https://github.com/kehao-chen)
+- [Kevin Huang](https://github.com/KevinSpringfield)
+- [Wei-Hsiang (Matt) Wang](https://github.com/mattwang44)
+- [roseyangwc](https://github.com/roseyangwc)
+- [Rosa Wu](https://github.com/rosawu159)
+- [tico88612](https://github.com/tico88612)
+- [Blocka](https://github.com/win5923)
+
+## 專案記事:
```
Comment (tico88612) [inline blog/glossary-zh-tw-first-published-2023.md:38]:
```suggestion
## 專案記事
```

File: blog/glossary-zh-tw-first-published-2023.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+### 1. 團隊建立(2023/04/26):
```
Comment (tico88612) [inline blog/glossary-zh-tw-first-published-2023.md:40]:
```suggestion
### 1. 團隊建立(2023/04/26)
```

File: blog/glossary-zh-tw-first-published-2023.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+    由小飛機[黃秉鈞(Phil Huang)](https://github.com/pichuang)發起建立團隊活動，
+    一開始Traditional Chinese Localization Team ，由[黃秉鈞(Phil Huang)](https://github.com/pichuang)和[傅耀德(Ader Fu)](https://github.com/ydFu)擔任初始 Approver，偕同 [Cody Chen](https://github.com/cdxvy30)等 3 位 volunteer 進行籌備。
+    - 建置團隊: https://github.com/cncf/glossary/issues/1987
+
+### 2. 專案啟動與專案管理(2023/05/04):
```
Comment (tico88612) [inline blog/glossary-zh-tw-first-published-2023.md:45]:
```suggestion
### 2. 專案啟動與專案管理(2023/05/04)
```

File: blog/glossary-zh-tw-first-published-2023.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+    依照 ISO-639 與 ISO-3166 確定語系使用 `zh-tw`後，由[傅耀德(Ader Fu)](https://github.com/ydFu)建立的完成專案本地化的初始設定和首次內容提交，並且創建繁體中文專用的 dev-tw branch，其中也經由 [Kay Lin](https://github.com/kaiyee0)建議下啟用 GitHub 內建的 Project 來追蹤本專案的現況。
+    - 本地化的初始設定和首次內容提交: https://github.com/cncf/glossary/pull/2006
+    - 使用 GitHub Project 管理: https://cloud-native.slack.com/archives/C055KERTNCX/p1691299769462459
+
+### 3. 繁中 Approver 團隊增加新成員(2023/09/01):
```
Comment (tico88612) [inline blog/glossary-zh-tw-first-published-2023.md:50]:
```suggestion
### 3. 繁中 Approver 團隊增加新成員(2023/09/01)
```

File: blog/glossary-zh-tw-first-published-2023.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+    由於當前貢獻者在繁體中文貢獻有著傑出的表現，
+    [邱宏瑋(HungWei Chiu)](https://github.com/hwchiu)在 CNCF 領域擁有深厚的知識和豐富的經驗，
+    [林自均(John Lin)](https://github.com/johnlinp)也在dev-tw 社群中，並提交許多貢獻與審核，
+    因此[黃秉鈞(Phil Huang)](https://github.com/pichuang)提名上述兩位加入繁體中文 approver 團隊，目前繁中 Approver 主要成員為[傅耀德(Ader Fu)](https://github.com/ydFu)、[黃秉鈞(Phil Huang)](https://github.com/pichuang)、[邱宏瑋(HungWei Chiu)](https://github.com/hwchiu)和[林自均(John Lin)](https://github.com/johnlinp)。
+    - 新增 approver :https://github.com/cncf/glossary/pull/2343
+
+### 4. 貢獻特別活動(2023/10/11):
```
Comment (tico88612) [inline blog/glossary-zh-tw-first-published-2023.md:57]:
```suggestion
### 4. 貢獻特別活動(2023/10/11)
```

File: blog/glossary-zh-tw-first-published-2023.md [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+    由矽谷牛的耕田筆記發起的貢獻 CNCF Glossary 可以換 Modern Web Conference (MWC) 票卷的特別活動，藉此累積不少新進貢獻者。
+    - 特別活動: https://www.facebook.com/100025250046893/posts/pfbid02VNAGgRaiZYJ3a6Ldjr3SCaGEDuL5F6bnxAaK6TkQp5Ghv5f8zos5ZW5zgHv68p6Rl/?mibextid=WiMSqg
+
+### 5. 繁體中文內容正式上線(2023/11/03):
```
Comment (tico88612) [inline blog/glossary-zh-tw-first-published-2023.md:61]:
```suggestion
### 5. 繁體中文內容正式上線(2023/11/03)
```

File: blog/glossary-zh-tw-first-published-2023.md [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+    由[傅耀德(Ader Fu)](https://github.com/ydFu)完成 28 個小項目的上線前準備，其中準備了 46 個繁體中文內容，並於經由 CNCF maintainer 審核後，內容正式上線並且發布在官方 CNCF glossary 網站中。
+    - 上線準備項目: https://github.com/cncf/glossary/issues/2355
+    - 繁體中文首次發布: https://github.com/cncf/glossary/pull/2576
+    - CNCF glossary繁體中文內容: https://glossary.cncf.io/zh-tw/#acknowledgements
+
+## 致謝:
```
Comment (tico88612) [inline blog/glossary-zh-tw-first-published-2023.md:67]:
```suggestion
## 致謝
```

File: blog/glossary-zh-tw-first-published-2023.md [hunk 10]
```diff
@@ -0,0 +1,72 @@
...
+謝謝大家的努力，中文本地化的工作仍在進行，歡迎舊雨新知共襄盛舉!!一起努力!!😀
+
+同步訊息:
+- [CNTUG Facebook](https://www.facebook.com/groups/cloudnative.tw/permalink/1719077178595610/?mibextid=oMANbw)
```
Comment (tico88612) [inline blog/glossary-zh-tw-first-published-2023.md:71]: 追蹤碼資訊去除
```suggestion
- [CNTUG Facebook](https://www.facebook.com/groups/cloudnative.tw/permalink/1719077178595610)
```

**Review by tico88612 — COMMENTED** [review]
標題應該不用加上冒號，TOC 呈現會比較好看。

File: blog/glossary-zh-tw-first-published-2023.md (hunk 10 above)
Comment (ydFu) [reply blog/glossary-zh-tw-first-published-2023.md]: thanks

**Review by pichuang — APPROVED** [review]
Approve, and merge the pr.

**Pull request merged**